README.md

```diff
@@ -19,6 +19,18 @@ Publish HASS events to your [Elasticsearch](https://elastic.co) cluster!
 - [Home Assistant Community Store](https://github.com/custom-components/hacs)
 - Home Assistant 2024.1
 
+The following table covers the Elasticsearch functionality used by the integration when configured against various versions of Elasticsearch:

```

Comment:
Can we update this statement too?
https://github.com/strawgate/homeassistant-elasticsearch/blob/6581837ed2a854032e5b29a135f103e3c15f96fa/README.md#L170